.moai/project/codemaps/modules.md

```
@@ -153,6 +153,12 @@
 **핵심**: `moai goal arm|status|clear`, Condition {Mechanical,Model}, Stop-hook 평가 계약
 **상태**: `.moai/state/goal/<session-id>.json` (세션별)
 
+### internal/factory
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)